src/pages/AddRecord/AddRecordColor.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react'
+import { ReactComponent as Check } from '@assets/check.svg'
+
+function AddRecordColor() {
+  const [colors, setColors] = useState([
+    { src: 'bg-icon-purple', choosed: true, id: 0 },
+    { src: 'bg-icon-yellow', choosed: false, id: 1 },
+    { src: 'bg-icon-pink', choosed: false, id: 2 },
+    { src: 'bg-icon-blue', choosed: false, id: 3 },
+    { src: 'bg-icon-green', choosed: false, id: 4 },
+  ])
+
+  const handleCurrentColorChoosing = (index: number): void => {
+    const changeCurrent = colors.map((color, currentIndex) => {
+      if (currentIndex === index) return { ...color, choosed: true }
+      return { ...color, choosed: false }
+    })
```

Comment:
여기는 이렇게 바꿀 수 있을 것 같습니당!

```suggestion
    const changeCurrent = colors.map((color, currentIndex) => ({
      ...color,
      choosed: currentIndex === index,
    }))
```

Reply:
오호 이렇게하면 코드수도 훨씬 줄고 더 좋은거같아요 반영하겠습니다!